Game/src/Scenes/GameplayScene.cpp

```diff
@@ -208,7 +208,7 @@ void GameplayScene::Render(double delta) {
 
 	if (m_drawCombo && std::get<7>(scores) > 0) {
 		m_amplitude = 30.0;
-		m_wiggleTime = 24.0 * m_comboTimer;
+		m_wiggleTime = 25.0 * m_comboTimer;
```

Comment:
you can adjust m_wiggleTime from 24 to 30 (or maybe on new engine it should be good at 30) otherwise animation will became weird